src/SSW.Rewards.Domain/Entities/CompletedQuiz.cs

```diff
@@ -5,4 +5,6 @@ public class CompletedQuiz : BaseEntity
     public virtual User User { get; set; }
     public int QuizId { get; set; }
     public virtual Quiz Quiz { get; set; }
+    public bool Passed { get; set; }
+    public List<QuizAnswer> Answers { get; set; } = new();
```

Comment:
This won't work - the QuizAnswer table is for Quizzes, not for user-submitted answer tracking. We will need a new table for this.

Reply:
Done